py-wacz/wacz/waczindexer.py

```diff
@@ -230,7 +264,6 @@ def serialize_json_pages(self, pages):
             yield json.dumps(data) + "\n"
 
     def generate_metadata(self, res, wacz):
-
         package_dict = {}
```

Comment:
can you remove this line if you're going to change the name to metadata

Reply:
Not sure i understand. the `package_dict= {}` line? It is still using package_dict as the root of the JSON, then just setting `package_dict["resources"]` and `package_dict["metadata"]`